lib/node/RequireChunkLoadingRuntimeModule.js

```diff
@@ -35,6 +36,32 @@ class RequireChunkLoadingRuntimeModule extends RuntimeModule {
 		const hasJsMatcher = compileBooleanMatcher(
 			chunkGraph.getChunkConditionMap(chunk, chunkHasJs)
 		);
+
+		let rootOutputDir = "'.'";
+		try {
+			if (chunk.hasEntryModule()) {
+				const outputName = this.compilation.hooks.assetPath.call(
+					this.compilation.outputOptions.filename + "",
+					{
+						chunk,
+						hash: chunk.hash
+					},
+					{}
+				);
+				const chunkInnerDirsCount = outputName.split(path.sep).length - 1;
```

Comment:
```suggestion
				const chunkInnerDirsCount = outputName.split("/").length - 1;
```

Regarding `path.sep`, good intention, but in this case it's not correct. These paths are "relative requests" which always use "/".